### PR TITLE
doc: update sample explaning "%" operator in test suites

### PR DESCRIPTION
**Pull request by tchaikov**
Signed-off-by: Kefu Chai <kchai@redhat.com>

### CODE REVIEW

File: doc/dev/index.rst [hunk 1]
```diff
@@ -1132,11 +1132,12 @@ Reducing the number of tests
...
 The ``rados`` suite generates thousands of tests out of a few hundred
-files. For instance, all tests in the `rados/thrash suite
-<https://github.com/ceph/ceph/tree/master/qa/suites/rados/thrash>`_
-run for ``xfs``, ``btrfs`` and ``ext4`` because they are combined (via
-special file ``%``) with the `fs directory
-<https://github.com/ceph/ceph/tree/master/qa/suites/rados/thrash/fs>`_
+files. For instance, all tests in the `rados/basic suite
+<https://github.com/ceph/ceph/tree/master/qa/suites/rados/basic>`_
+run with different messenger types: ``simple``, ``async`` and
+``random``, because they are combined (via special file ``%``) with the
```
Comment (smithfarm) [inline doc/dev/index.rst]: s/via special/via the special/

File: doc/dev/index.rst [hunk 2]
```diff
@@ -1132,11 +1132,12 @@ Reducing the number of tests
 ----------------------------
 
 The ``rados`` suite generates thousands of tests out of a few hundred
-files. For instance, all tests in the `rados/thrash suite
```
Comment (smithfarm) [inline doc/dev/index.rst]: This happens because teuthology constructs test matrices from subdirectories wherever it encounters a file named ``%``.

Comment (smithfarm) [inline doc/dev/index.rst]: Suggest to insert new sentence after "files." and before "For instance,"

**Comment by tchaikov** [comment]
@smithfarm fixed and repushed.

**Pull request merged**